docs/source/getting_started.rst

```diff
@@ -50,18 +51,25 @@ With the activated ``conda`` environment, you can install with the
 ::
 
    (figshare_patrons) $ cd /path/to/parent/folder/ReQUIAM
-   (figshare_patrons) $ (sudo) python setup.py develop
+   (figshare_patrons) $ python setup.py develop
 
 This will automatically installed the required ``pandas``, ``ldap3``,
 ``requests``, and ``numpy`` packages.
 
 You can confirm installation via ``conda list``
+:: 
+   (figshare_patrons) $ conda list 
```

Comment:
Please fix the indentation on line 61 to address the warning from the Sphinx Doc Checker